README.md

```diff
@@ -1,62 +1,65 @@
 # The Node.js Foundation TSC
 
-The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
+This is the primary communication and governance hub of the Node.js
+Foundation's Technical Steering Committee. Its goals are:
 
-For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
+* Maintaining, releasing and improving a healthy Node.js Core project.
+* Fostering and encouraging a healthy Node.js open source ecosystem.
```

Comment:
Ugh. I hate to dive in to such a nit-pick, sorry in advance to everyone....

Line 6 is the one that is a bit off. Adding `health` makes it the subject of the actions (which I believe is correct & important). You can `maintain` and `improve` health- but not `release` it. I believe `release` is intended to have Node.js Core (code) as the subject- which obviously you can release code. Careful guidance & oversight of the releases of Node.js Core releases will maintain and improve it. I'm sure everyone will 'get the idea' and probably just read past- so that's why I just said "consider rewording."